File: result/translate/chatgpt/python2java/java/SUM_FACTORS_NUMBER.java
public static int f_gold(int n) {
    int result = 0;
    for (int i = 2; i <= (int) Math.sqrt(n); i++) {
        if (n % i == 0) {
            if (i == (n / i)) {
                result = result + i;
            } else {
                result = result + (i + n / i);
            }
        }
    }
    return (result + n + 1);
}

File: result/translate/chatgpt/java2python/python/SUM_FACTORS_NUMBER.py
def f_gold(n):
    result = 0
    i = 2
    while i <= int(n**0.5):
        if n % i == 0:
            if i == (n // i):
                result += i
            else:
                result += (i + n // i)
        i += 1
    return (result + n + 1)